### ISSUE TITLE
Exception in Wkhtml2pdf.php line 888

### ISSUE THREAD

**Issue by givre** [Attendize/Attendize#219]
Hi, 
I got this error on a new server ( centos 6 ) when I try to register a new ticket. 
The system get an error 500 when sending POST request after I click to "Register" 

```
WKHTMLTOPDF didn't return any data
in Wkhtml2pdf.php line 888
at Wkhtml2pdf->_render() in Wkhtml2pdf.php line 1047
at Wkhtml2pdf->output('F', '[...]/public/user_content/pdf_tickets/AJTKZ1310.pdf') in Wkhtml2pdf.php line 243
```
Any idea about this ?  
Thanks

**Comment by jharris70** [comment]
I'm having this issue also with line 888, but only when creating an order with more than around 25 tickets. I think Wkhtml2pdf is running out of memory, but I don't know how to track it down yet.

**Comment by givre** [comment]
> I'm having this issue also with line 888, but only when creating an order with more than around 25 tickets. I think Wkhtml2pdf is running out of memory, but I don't know how to track it down yet.

Hum strange. It happens to me with any orders on my new system . I should check the memory  and get back to you. 

You think it can be about php value : memory_limit ?

**Comment by jharris70** [comment]
I tried a couple of things when I got the error last night. I was trying to print 100 consignment tickets for my group when I first saw it.

I modified the line to include the ````$command```` in the error log and was able to run the command from shell.

For me, placing an order of fewer tickets allowed Wkhtml2pdf to operate correctly, although I am getting an error occurred message at the top end, but I'm still getting an email with the unpaid tickets (offline ordering is enabled).

This leads me to believe that Wkhtml2pdf doesn't have proper error handling for running out of memory.
